### DISCUSSION TITLE
Increase font size of result

### DISCUSSION THREAD

**Discussion by CRAG666** [tridactyl/tridactyl#4508] [Q&A]
![imagen](https://user-images.githubusercontent.com/34254373/207747378-3320e1d3-5fc8-4299-8cfb-6122af0dd67b.png)
I would like to change the source size of the results for a larger size because I fail to see the results

**Comment by bovine3dom** [comment]
Sorry, I missed this. I think if you change some of these variables https://github.com/tridactyl/tridactyl/blob/c663b8e23a5f3125ad0a28672f993497756f9f7c/src/static/themes/default/default.css#L6 in userContent.css http://kb.mozillazine.org/UserContent.css it should work
